src/package_io/input_parser.star

```diff
@@ -442,6 +443,7 @@ def get_default_mev_params():
         "mev_builder_image": "flashbots/builder:latest",
         "mev_builder_cl_image": "sigp/lighthouse:latest",
         "mev_boost_image": "flashbots/mev-boost",
+        "mev_boost_args": ["mev-boost", "--relay-check"],
```

Comment:
why is `mev-boost` an arg to MEV?

Reply:
i believe `mev-boost` should be the entry point / default command and everything else should be extra

Reply:
it is the case for the flashbots impl. I am unsure about other impls. I ll check out @parithosh mentioned mev-sharp